shlr/java/class.c

```diff
@@ -726,97 +665,135 @@ RZ_API DsoJsonObj *rz_bin_java_get_class_info_json(RzBinJavaObj *bin) {
 		bool is_annotation = ((class_->visibility & RZ_BIN_JAVA_CLASS_ACC_ANNOTATION) != 0);
 		bool is_enum = ((class_->visibility & RZ_BIN_JAVA_CLASS_ACC_ENUM) != 0);
 
-		dso_json_dict_insert_str_key_num(class_info_dict, "access_flags", class_->visibility);
-		dso_json_dict_insert_str_key_num(class_info_dict, "is_public", is_public);
-		dso_json_dict_insert_str_key_num(class_info_dict, "is_final", is_final);
-		dso_json_dict_insert_str_key_num(class_info_dict, "is_super", is_super);
-		dso_json_dict_insert_str_key_num(class_info_dict, "is_interface", is_interface);
-		dso_json_dict_insert_str_key_num(class_info_dict, "is_abstract", is_abstract);
-		dso_json_dict_insert_str_key_num(class_info_dict, "is_synthetic", is_synthetic);
-		dso_json_dict_insert_str_key_num(class_info_dict, "is_annotation", is_annotation);
-		dso_json_dict_insert_str_key_num(class_info_dict, "is_enum", is_enum);
-		dso_json_dict_insert_str_key_str(class_info_dict, "name", class_->name);
-
-		if (!class_->super) {
-			DsoJsonObj *str = dso_json_str_new();
-			dso_json_dict_insert_str_key_obj(class_info_dict, "super", str);
-			dso_json_str_free(str);
-		} else {
-			dso_json_dict_insert_str_key_str(class_info_dict, "super", class_->super);
-		}
+		pj_kb(j, "access_flags", class_->visibility);
```

Comment:
`pj_ki`?

Reply:
yes, i noticed i made a mistake. pj_kN should be better.